examples/kotlin-multiplatform/build.gradle

```diff
@@ -105,6 +107,10 @@ benchmark {
         register("jvm") {
```

Comment:
The comment above seems to be outdated as the name now should be a target name or a compilation default source set name